programs/dynamic-bonding-curve/src/instructions/migration/meteora_damm/meteora_damm_claim_lp_fee.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{const_pda, state::MigrationProgress, *};
+
+pub fn handle_migrate_meteora_damm_claim_lp_fee_token<'info>(
+    ctx: Context<'_, '_, '_, 'info, MigrateMeteoraDammClaimLpTokenCtx<'info>>,
```

Comment:
I feel a bit redundant for the action, because the claim lp_token would work as the same 
How about to force transaction to lock lp firstly, then claim_lp token would claim both lp token and the lp token as fee

Reply:
Not sure is there any public migrator that do `claim_lp`, then only `lock_lp`. This could break them.

Reply:
Fixed